pkg/live/inventoryrg.go

```diff
@@ -30,16 +29,16 @@ import (
 // the Inventory interface. This wrapper loads and stores the
 // object metadata (inventory) to and from the wrapped ResourceGroup.
 type InventoryResourceGroup struct {
-	inv      *resource.Info
+	inv      *unstructured.Unstructured
```

Comment:
unstructured.Unstructured doesn't convey much information but looks like it is from k8s library.